README.md

```diff
@@ -143,18 +143,20 @@ params.MinSplitSize = 7168
 reader, totalDownloaded, totalExpected, err := rtManager.DownloadFilesWithResultReader(params)
 ```
 Use `reader.NextRecord()` and `FileInfo` type from `utils` package to iterate over the download results.
-Use `reader.Close()` method (preferably using `defer`), to remove the results reader after it is used:
-``` 
+```
 defer reader.Close()
-var file utils.FileInfo
-for e := resultReader.NextRecord(&file); e == nil; e = resultReader.NextRecord(&file) {
+for file := new(utils.FileInfo); resultReader.NextRecord(file) == nil; file = new(utils.FileInfo) {
```

Comment:
The usage of the reader, including the iteration and reset is not relevant for the download itself. People can read about it dedicated section.

Reply:
Removed.